pkg/ipam/ipam_linux.go

```diff
@@ -117,6 +117,7 @@ func ConfigureIface(ifName string, res *current.Result) error {
 			Dst:       &r.Dst,
 			LinkIndex: link.Attrs().Index,
 			Gw:        gw,
+			Priority:  r.Priority,
```

Comment:
The error could return the priority `return fmt.Errorf("failed to add route '%v via...`